ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainService.kt

```diff
@@ -73,6 +77,7 @@ class MainService : Service() {
    }
 
     override fun onCreate() {
+        initNNStreamer()
```

Comment:
should we abort the process if failed to initialize nnstreamer and native functions? (this.initialized)

Reply:
I added log and `stopSelf()` fuction.